featuremanagement/_defaultfilters.py

```diff
@@ -11,7 +11,7 @@
 
 from ._featurefilters import FeatureFilter
 
-FEATURE_FLAG_NAME_KEY = "name"
+FEATURE_FLAG_NAME_KEY = "feature_name"
```

Comment:
I'm not super familiar with this library, but why is this key getting changed? Did the `context` variable structure change?

Reply:
Gotcha, thanks